sklearn/utils/estimator_checks.py

```diff
@@ -597,7 +598,7 @@ def check_fit2d_1sample(name, estimator_orig):
 
 @ignore_warnings
 def check_fit2d_1feature(name, estimator_orig):
-    # check by fitting a 2d array and prediting with a 1d array
+    # check by fitting a 2d array with only one feature
     rnd = np.random.RandomState(0)
     X = 3 * rnd.uniform(size=(10, 1))
     y = X[:, 0].astype(np.int)
```

Comment:
Using the data set generated in `check_fit2d_1feature` the following estimators return a `ValueError`:
- FeatureAgglomeration, ValueError('Found array with 1 feature(s) (shape=(10, 1)) while a minimum of 2 is required by FeatureAgglomeration.',)
- GraphLasso, ValueError('Found array with 1 feature(s) (shape=(10, 1)) while a minimum of 2 is required by GraphLasso.',)
- GraphLassoCV, ValueError('Found array with 1 feature(s) (shape=(10, 1)) while a minimum of 2 is required by GraphLassoCV.',)
- OrthogonalMatchingPursuitCV, ValueError('Found array with 1 feature(s) (shape=(10, 1)) while a minimum of 2 is required by OrthogonalMatchingPursuitCV.',)
- RANSACRegressor, ValueError('RANSAC could not find a valid consensus set. All `max_trials` iterations were skipped because each randomly chosen sub-sample failed the passing criteria. See estimator attributes for diagnostics (n_skips*).',)
- RandomizedLogisticRegression, ValueError('This solver needs samples of at least 2 classes in the data, but the data contains only one class: 1',)
- SelectKBest, ValueError("k should be >=0, <= n_features; got 10.Use k='all' to return all features.",)

If `k='all'`, `SelectKBest` does not raise any exception. For `RandomizedLogisticRegression`, the `ValueError` is raised because one of the random subsamples of the data set contains only 1 class. I don't know how this should be handled.